src/js/utils/ErrorMessageUtil.js

```diff
@@ -0,0 +1,41 @@
+const MESSAGE_VARIABLE = /\{\{([^}]+)\}\}/g;
```

Comment:
It was initially one brace, then I change it to two braces, but why now you think three is better? I seriously doubt an expression with two braces will appear on an error message...